api-audit/src/main/java/com/capitalone/dashboard/evaluator/StaticSecurityAnalysisEvaluator.java

```diff
@@ -63,16 +63,14 @@ public SecurityReviewAuditResponse evaluate(CollectorItem collectorItem, long be
     private SecurityReviewAuditResponse getStaticSecurityScanResponse(CollectorItem collectorItem, long beginDate, long endDate) {
         List<CodeQuality> codeQualities = codeQualityRepository.findByCollectorItemIdAndTimestampIsBetweenOrderByTimestampDesc(collectorItem.getId(), beginDate - 1, endDate + 1);
 
-        ObjectMapper mapper = new ObjectMapper();
         SecurityReviewAuditResponse securityReviewAuditResponse = new SecurityReviewAuditResponse();
-
         if (CollectionUtils.isEmpty(codeQualities)) {
             securityReviewAuditResponse.addAuditStatus(CodeQualityAuditStatus.STATIC_SECURITY_SCAN_MISSING);
             return securityReviewAuditResponse;
         }
 
         CodeQuality returnQuality = codeQualities.get(0);
-        securityReviewAuditResponse.setUrl(collectorItem.getOptions().get(STR_REPORT_URL).toString());
+        securityReviewAuditResponse.setAuditEntity(collectorItem.getOptions());
```

Comment:
We should set setAuditEntity and setLastExecutionTime before ` if (CollectionUtils.isEmpty(codeQualities))`. That way the user still gets some feedback about the configured widget.

Reply:
Also, can you set the `lastUpdated` field.

Reply:
@Sbrenthughes auditEntity is assigned before empty check, added lastUpdated field as well